torch/csrc/utils/python_dispatch.cpp

```diff
@@ -479,14 +479,14 @@ void initDispatchBindings(PyObject* module) {
 
 #define DEF_ONE(n) .value(#n, c10::DispatchKey::n)
 
-  py::enum_<c10::DispatchKey>(m, "DispatchKey") DEF_ONE(Undefined)
-      DEF_ONE(CompositeExplicitAutogradNonFunctional)
-          DEF_ONE(CompositeExplicitAutograd)
-              DEF_ONE(CompositeImplicitAutogradNestedTensor)
-                  DEF_ONE(CompositeImplicitAutograd) DEF_ONE(AutogradOther)
-                      DEF_ONE(Autograd) DEF_ONE(BackendSelect)
-                          DEF_ONE(ADInplaceOrView) DEF_ONE(PythonTLSSnapshot)
-                              DEF_ONE(Python)
+  py::enum_<c10::DispatchKey>(m, "DispatchKey") DEF_ONE(Undefined) DEF_ONE(
+      CompositeExplicitAutogradNonFunctional) DEF_ONE(CompositeExplicitAutograd)
+      DEF_ONE(CompositeImplicitAutogradNestedTensor)
+          DEF_ONE(CompositeImplicitAutograd) DEF_ONE(AutogradOther)
+              DEF_ONE(Autograd) DEF_ONE(BackendSelect) DEF_ONE(ADInplaceOrView)
+                  DEF_ONE(PythonTLSSnapshot) DEF_ONE(Python)
+                      DEF_ONE(FuncTorchDynamicLayerFrontMode)
+                          DEF_ONE(FuncTorchDynamicLayerBackMode)
```

Comment:
Do we need to support back mode now?

Reply:
We do not use it in this PR. I'm adding it here so that it is exposed to Python so we don't need to remember to expose it in the future. Ideally torch._C.DispatchKey has exactly everything that c++ DispatchKey contains, but it doesn't because people update it lazily